ruamel_yaml/meta.yaml

```diff
@@ -29,4 +31,6 @@ test:
 about:
   home: https://bitbucket.org/ruamel/yaml
   license: MIT
+  license_family: MIT
```

Comment:
Good to know.

Any chance we can get this or any of these other fields documented? Right now the [`about` docs]( http://conda.pydata.org/docs/building/meta-yaml.html#about-section ) only really mention `license_file`, but nothing about `license_family`, `doc_url`, or `dev_url`. It would also be good to know if there are any other special cases like this (e.g. BSD?).